final_prep/README.md

```diff
@@ -19,37 +19,43 @@ Some things to consider as you refactor include...
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [X] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [X] Study/Project work time
+- [X] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+- <img width="1213" alt="Screen Shot 2021-08-07 at 2 44 21 AM" src="https://user-images.githubusercontent.com/75279942/128594731-33443550-4e03-426e-97a6-0ffea8abcaf9.png">
+
 - `Add Week 2 Screenshot Here`
+- <img width="1165" alt="Screen Shot 2021-08-07 at 2 44 28 AM" src="https://user-images.githubusercontent.com/75279942/128594741-8212bf17-02b5-4d8a-928d-e9da249e8a27.png">
+
 - `Add Week 3 Screenshot Here`
+- <img width="1184" alt="Screen Shot 2021-08-07 at 2 44 35 AM" src="https://user-images.githubusercontent.com/75279942/128594746-4757ca98-6a71-4099-a590-171518500147.png">
+
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [X] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [X] Add link to your gist here: https://gist.github.com/Henchworm/bf4641b1b806bdae33547dfa7b4834b2
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [X] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [X] Add a link to your gist here: https://gist.github.com/Henchworm/19b7565bba7516cab0744c96c2b5d3ad
 
 ### Group Work Prep
 As part of Turing's project-based learning approach, you will often be working in pairs or larger groups. In order to set yourself (and your team) up for success, it is important to ensure you are prepared to be an equitable contributor and teammate.
-- [ ] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
-  - [ ] Add a link to your gist here: 
+- [X] Complete the [DTR Guiding Questions](https://gist.github.com/ericweissman/c56f3a98cdce761808c21d498a52f5c6)
+  - [X] Add a link to your gist here: https://gist.github.com/Henchworm/dcc7120886e4051d8abac4b5057fab41
```

Comment:
Nice self awareness here- I am like you, and prefer speaking on things rather than constant slack chatter. However, any work place will have a need for communication through systems like slack, github, workflow systems... There is plenty of opportunity to collaborate with your team through talking, but I do encourage practicing some communication that is outside your comfort level. These are the ways we grow at Turing!